thumbor/loaders/http_loader.py

```diff
@@ -11,30 +11,33 @@
 import datetime
 import re
 from typing import Pattern
-from functools import partial
+from urllib.parse import quote, unquote, urlparse
 
+import tornado.curl_httpclient
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.curl_httpclient'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)